test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/res/SampleTheme.xaml

```diff
@@ -5,12 +5,15 @@
   xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
   Id="MyTheme"
   BasedOn="Tizen.NUI.Theme.Common">
-
+    
+    <Theme.Resources>
+        <ResourceDictionary Source="{ApplicationResourcePath SampleThemeResourceDefault.xaml}"/>
+    </Theme.Resources>
 
     <!--Button-->
     <c:ButtonStyle x:Key="Tizen.NUI.Components.Button" >
         <c:ButtonStyle.BackgroundColor>
-            <Selector x:TypeArguments="Color" Normal="{x:StaticResource ButtonBackgroundColorNormal}" Pressed="{x:StaticResource ButtonBackgroundColorPressed}" Disabled="{x:StaticResource ButtonBackgroundColorDisabled}"/>
+            <Selector x:TypeArguments="Color" Normal="{ThemeResource ButtonBackgroundColorNormal}" Pressed="{ThemeResource ButtonBackgroundColorPressed}" Disabled="{ThemeResource ButtonBackgroundColorDisabled}"/>
```

Comment:
(b^_^)b 
you did great job! 
in the past, I had thought we must add and use this markup-extension into NUI XAML to expand the functionality.
Theoretically, we can add various and complex functionality using this markup-extension.
I am very happy that you have done this job that I couldn't finish.

Reply:
Since we don't want to support theme resource for xaml file yet, Theme Resource Markup Extension is removed.
Instead of Theme Resource Markup Extension, sample uses Dictionary for changed resources that will be generated from Tizen Theme Manager(https://github.com/Samsung/TizenFX/tree/master/internals/src/Tizen.Applications.ThemeManager).